tests/test_graphql_client.py

```diff
@@ -114,7 +114,7 @@ def test_execute_query_with_options(self, post_mock):
             "http://www.test-api.com/",
             json={"query": query},
             headers={},
-            auth=HTTPBasicAuth("fake@example.com", "not_a_real_password"),
+            auth=auth,
```

Comment:
Thanks for fixing this but how about we drop support for request v1? I feel like that would be more generous than needed because it's never been supported since this library was created.

Reply:
The last v1 release of requests came out in 2013, so I think that's fair. I was trying to super flexible here with what versions we allow, but in reality this is probably unecessary.

Reply:
I don't feel strongly about this. I'll leave it up to you. 😃